handler/error_handler_test.go

```diff
@@ -0,0 +1,123 @@
+package handler
+
+import (
+	"testing"
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/ONSdigital/dp-dimension-importer/mocks"
+	"errors"
+	"github.com/ONSdigital/dp-dimension-importer/event"
+	"time"
+	"github.com/ONSdigital/dp-dimension-importer/schema"
+	"sync"
+	"github.com/ONSdigital/go-ns/log"
+)
+
+const (
+	instance_id      = "0123456789"
+	valor_morghullis = "Valar morghulis"
+)
+
+var (
+	errorEvent = event.ErrorEvent{
+		InstanceID: instance_id,
+		EventType:  errEventType,
+		EventMsg:   valor_morghullis,
+	}
+
+	expectedErr = errors.New(valor_morghullis)
+)
+
+func TestErrorHandler_Handle(t *testing.T) {
+	Convey("Given errorHandler has been configured correctly", t, func() {
+
+		output := make(chan []byte)
```

Comment:
Consider using a channel with a buffer. This will allow the message to be sent to the channel without it blocking until something receives it. It should prevent the need to have a separate go routine and the select statement.